determined-ai/determined
# fix: sandbox experiment list settings for each project [WEB-129]

## Description

currently project details settings (filters, sorter, columns, pagination) are applied to all projects whenever they are applied to one. 

this is undesirable and causes issues esp when the pagination setting results in no data being displayed and an inability to turn the page.

## Test Plan

- navigate to a project
- click on uncategorized. click back. this will be your "two projects" 
- change filters sorters column ordering on one.
- go to the other
- very that the changes were not applied to the other project
- refresh 
- verify again
- go back
- verify the changes are still applied to the initial project.
etc.

## Commentary (optional)
duplication of `ProjectDetails.settings` as `ExperimentList.settings` had already caused some erroneous discrepancies, so i consolidated them.

for this one, i took the current `OldProjectDetails.settings` (i.e., the non-feature-flagged one) as the source of truth,  pasted it into `ExperimentList.settings`, and then made the modifications from there.

when comparing `ExperimentList.settings` and `OldProjectDetails.settings`, there were a couple of discrepancies, i discussed how i resolved them in PR comments below


## Checklist
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
